parser-service/pom.xml

```diff
@@ -40,12 +40,6 @@
             <groupId>org.mule.apikit</groupId>
             <artifactId>raml-parser-interface-impl-amf</artifactId>
             <version>${project.version}</version>
-            <exclusions>
```

Comment:
Why is this not required anymore?

Reply:
Is fixed on AMF 5 we had [a work](https://gus.lightning.force.com/lightning/r/a07EE00000wrF3qYAE/view) about this